packages/cardhost/tests/acceptance/create-card-test.js

```diff
@@ -151,11 +113,36 @@ module('Acceptance | card create', function(hooks) {
     await percySnapshot([assert.test.module.name, assert.test.testName, 'data-entered'].join(' | '));
   });
 
+  test('creating a card from the homepage', async function(assert) {
+    await login();
+    await visit('/');
+
+    assert.equal(currentURL(), '/');
+    await percySnapshot(assert);
+    await click('[data-test-new-blank-card-btn]');
+    await setCardName(card1Name);
+
```

Comment:
- I think we should add a percy snapshot that shows the open modal.
- You could also add the open modal view to the `accessibility-test.js`
- maybe we need a test that also clicks the close button and asserts that the modal is gone, unless that is already tested under the dialog component